app/assets/javascripts/views/story_view.js

```diff
@@ -499,6 +512,20 @@ module.exports = FormView.extend({
       );
     }
 
+    const attachments = this.$('.story-attachments')[0];
+    if(attachments) {
+      ReactDOM.render(
+        <StoryAttachment
+          name={'attachments'}
+          isReadonly={this.isReadonly()}
+          files={this.model.get('documents')}
+          attachinaryProps={() => this.makeAttachinaryProps()}
```

Comment:
Looks like you're not using `this` inside `makeAttachinaryProps`, being to you can do just:

```js
attachinaryProps={this. makeAttachinaryProps}
```